src/Avalonia.Visuals/Rendering/DirtyVisuals.cs

```diff
@@ -68,19 +79,19 @@ public void Clear()
         /// Removes a visual from the dirty list.
         /// </summary>
         /// <param name="visual">The visual.</param>
-        /// <returns>True if the visual was present in the list; otherwise false.</returns>
-        public bool Remove(IVisual visual)
+        public void Remove(IVisual visual)
```

Comment:
`Remove` doesn't actually ever seem to be called... Maybe we should remove this method?